deploy/charts/cert-manager/README.md

```diff
@@ -77,6 +68,17 @@ $ helm delete my-release
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
+If you want to completely uninstall cert-manager from your cluster, you will also need to
+delete the previously installed CustomResourceDefinition resources:
+
+```console
+# Kubernetes 1.15+
+$ kubectl delete -f https://github.com/jetstack/cert-manager/releases/download/v0.14.0/cert-manager.yaml
```

Comment:
this is not the correct link for the CRDs I guess

Reply:
Nice catch!